content/courses/color-vision-colorimetry.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Color Vision and Colorimetry"
+image: "/images/courses/color.webp"
+weight: 5
+draft: false
+---
+
+**Course Duration:** 30 hours  
```

Comment:
maybe course duration and instructor could be part of the info in the frontmatter ?